ert_logging/logger.conf

```diff
@@ -21,7 +21,7 @@ handlers:
 loggers:
   ert_shared.storage:
     level: DEBUG
-    handlers: [apifile]
+    handlers: [console, apifile]
```

Comment:
Why console?

Reply:
Correct me if i'm wrong - but i think one of the reasons making a log file was to avoid flooding of the console. Now we both lower the bar with INFO and pushing to console which really makes a lot of messages. What i think was the purpose of this task was to get flask to print it's address and port ( which it did back in time at the same time it prints the other msgs ).  What i would have done is to get host and port flask is running at and print it. Not change log level nor adding console logger.

Reply:
The _Serving Storage API on http://........._ is present in your PR, but i don't want logging in console with INFO for the rest of the msgs.... 😬 

```
2020-09-15 09:10:49,678 - ert_shared.storage.autoclient - INFO - _Serving Storage API on http://127.0.0.1:64196_
 * Serving Flask app "ert http api" (lazy loading)
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: off
```